### PR TITLE
docs: Update `VibrationDevice` to include platform specific requirements

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): part of https://github.com/unoplatform/uno/issues/10614

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: doc/articles/features/windows-devices-haptics.md [hunk 1]
```diff
@@ -11,10 +11,22 @@ uid: Uno.Features.WDHaptics
...
 ## `VibrationDevice` class
 
-The `RequestAccessAsync` method is implemented on all platforms and returns `Allowed` on all platforms, except for Android and Tizen. In the case of Android, the `android.permission.VIBRATE` permission needs to be declared. In the case of Tizen, the `http://tizen.org/privilege/haptic` privilege needs to be declared.
+### Platform-specific requirements
 
 The `GetDefaultAsync` method is implemented on all platforms and returns `null` for the unsupported platforms (WPF, GTK).
 
+#### Android
+
+For Android, there are is one permission you must configure before using this API in your project. To do that, add the following to `AndroidManifest.xml`:
```
Comment (Youssef1313) [inline doc/articles/features/windows-devices-haptics.md:20]:
```suggestion
For Android, there is one permission you must configure before using this API in your project. To do that, add the following to `AndroidManifest.xml`:
```

File: doc/articles/features/windows-devices-haptics.md [hunk 2]
```diff
@@ -11,10 +11,22 @@ uid: Uno.Features.WDHaptics
...
+
+```xml
+<uses-permission android:name="android.permission.VIBRATE" />
+```
+
+#### Tizen
+
+In the case of Tizen, the `http://tizen.org/privilege/haptic` privilege needs to be declared in `config.xml` file.
```
Comment (Youssef1313) [inline doc/articles/features/windows-devices-haptics.md:28]:
```suggestion
For Tizen, the `http://tizen.org/privilege/haptic` privilege needs to be declared in `config.xml` file.
```

Comment (Youssef1313) [inline doc/articles/features/windows-devices-haptics.md:28]: Is it `config.xml` or `tizen-manifest.xml`?

Comment (morning4coffe-dev) [reply doc/articles/features/windows-devices-haptics.md]: I have never worked with Tizen before, so when I was looking at the docs, I found only config.xml, but when I search for the tizen-manifest.xml specifically, I get a lot of results, it's probably that one.

**Review by ajpinedam — APPROVED** [review]
Looks good. Thanks

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

**Pull request merged**